assets/stylesheets/style.scss

```diff
@@ -238,6 +238,9 @@
 @import 'components/popover/style';
 @import 'components/tooltip/style';
 
+// All WordPress Component styles
+@import '~@wordpress/components/build-style/style.css';
```

Comment:
Not sure why this change is showing here. It's already in develop, I added as part of my tooltip work, did you add it too? Probably you should merge develop and make sure this line is not duplicated.

Reply:
Yeah, I rebased my branch off yours so the merge will be easier. It shouldn't cause any problems.